.github/scripts/publish.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+echo "Trying to publish the package to npm for tag $RAW_TAG_NAME"
+
+# Validate tag's format follows conventions eg v00.01.02 or v0.90.2-alpha.5
```

Comment:
Leading zeroes aren't allowed in the version part of a semver tag: https://semver.org/#spec-item-2

I don't think our validation needs to be perfect here (the regex could get over-complicated if we try to mirror semver) but the comment shouldn't have `v00.01.02` as an example of a conventional tag.